3-Solving-Problems-By-Searching/poisson_distribution.js

```diff
@@ -0,0 +1,78 @@
+function poissonDistribution(height, width, radius) {
+  let sample = poissonDiscSampler(width, height, radius);
+  let nodes = [];
+  let s = sample();
+  let grid = [];
+  while (s) {
+    nodes.push(s.point);
+    grid = s.grid;
+    s = sample();
+  }
+  return [nodes, grid];
+}
+
+function poissonDiscSampler(width, height, radius) {
```

Comment:
I will add the license in the footer. Would that be fine ? Or should I try to implement my own version of the algorithm? 
Actually I tried that and for some reason couldn't get it right. After a complete day wasted on it, I decided to go for the implemented version.